Assets/_BForBoss/_Core/Scripts/Debug/DebugFreeRoamCamera.cs

```diff
@@ -0,0 +1,261 @@
+using System;
+using UnityEngine.InputSystem;
+using UnityEngine;
+using UnityEngine.InputSystem.LowLevel;
+
+namespace BForBoss
+{
+    public class DebugFreeRoamCamera : MonoBehaviour
```

Comment:
did you reference this class from somewhere else? 
can u post the link to that reference?

Reply:
Updated PR description with reference link! Note that it has been modified quite a bit